Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/PyYAML-3.12.egg-info'...
Listing './lib/python3.5/site-packages/aenum'...
Listing './lib/python3.5/site-packages/aenum-1.4.5.dist-info'...
Listing './lib/python3.5/site-packages/aioamqp'...
Listing './lib/python3.5/site-packages/aioamqp-0.10.0.dist-info'...
Listing './lib/python3.5/site-packages/h11'...
Listing './lib/python3.5/site-packages/h11/tests'...
Listing './lib/python3.5/site-packages/h11/tests/data'...
Listing './lib/python3.5/site-packages/h11-0.7.0.dist-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/waspy'...
Compiling './lib/python3.5/site-packages/waspy/router.py'...
***   File "./lib/python3.5/site-packages/waspy/router.py", line 169
    raise ValueError(f"Duplicate route exists {method}")
                                                      ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/waspy/transports'...
Compiling './lib/python3.5/site-packages/waspy/transports/httptoolstransport.py'...
***   File "./lib/python3.5/site-packages/waspy/transports/httptoolstransport.py", line 172
    print(f'-- Listening for HTTP on port {self.port} --')
                                                        ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/waspy/transports/httptransport.py'...
***   File "./lib/python3.5/site-packages/waspy/transports/httptransport.py", line 150
    logger.info(f'-- Listening for HTTP on port {self.port} --')
                                                              ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/waspy/transports/rabbitmqtransport.py'...
***   File "./lib/python3.5/site-packages/waspy/transports/rabbitmqtransport.py", line 79
    self._starting_future: asyncio.Future = asyncio.ensure_future(self.connect())
                         ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/waspy/webtypes.py'...
***   File "./lib/python3.5/site-packages/waspy/webtypes.py", line 131
    return (f'Request(headers={repr(self.headers)}, path={repr(self.path)}, '
                                                                            ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/waspy-0.22.0.dist-info'...
Listing './lib/python3.5/site-packages/yaml'...
Listing './share'...
Listing './share/python-wheels'...
